hash/a2600.xml

```diff
@@ -14787,7 +14787,7 @@ MOS Atari-made game NTSC ROMs had a CO16xxx number and PAL ROMs had CO17xxx numb
 	</software>
 
 	<software name="spacecave" cloneof="spacecav">
-		<description>Space Cavern - Les guerriers de l'espace (PAL)</description>
+		<description>Space Cavern - Les Guerriers de l'Espace (PAL)</description>
 		<year>1981</year>
```

Comment:
I think the capitalisation of the subtitle here intentionally uses sentence case to match the box.  See images of front and back here: https://www.atarimania.com/pgesoft.awp?version=12591

Reply:
You're correct. I undid this change.